packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -348,6 +353,30 @@ class GoogleMapsFlutterAndroid extends GoogleMapsFlutterPlatform {
     );
   }
 
+  @override
+  Future<void> updateGroundOverlays(
+    GroundOverlayUpdates groundOverlayUpdates, {
+    required int mapId,
+  }) {
+    assert(
+        groundOverlayUpdates.groundOverlaysToAdd.every(
+            (GroundOverlay groundOverlay) =>
+                groundOverlay.position == null || groundOverlay.width != null),
+        'On android width must be set when position is set for ground overlays.');
```

Comment:
Should it be called "Android" (capitalized) here?
```suggestion
        'On Android width must be set when position is set for ground overlays.');
```